setup.py

```diff
@@ -76,6 +77,5 @@ def finalize_options(self):
             'eventio_cut_file = eventio.scripts.cut_eventio_file:main',
         ]
     },
-    setup_requires=['pytest-runner', 'numpy'],
```

Comment:
pytest-runner will be just removed with this change, is it correct?

Reply:
Yes, `pytest-runner` enables doing `python setup.py test`, but this is not used anymore.